changelog/feature.7935.md

```diff
@@ -0,0 +1 @@
+IBM Watson Training Data Importer. This will allow to connvert IBM Watson V2 Training data downloadable from IBM Workspaces into a Rasa format.
```

Comment:
```suggestion
IBM Watson Training Data Importer. This will allow to convert IBM Watson V2 Training data downloadable from IBM Workspaces into a Rasa format.
```